code/game/objects/items/storage/boxes/job_boxes.dm

```diff
@@ -89,6 +89,10 @@
 	internal_type = /obj/item/tank/internals/emergency_oxygen/engi
 	medipen_type =  /obj/item/reagent_containers/hypospray/medipen/atropine
 
+/obj/item/storage/box/survival/syndie/Initialize(mapload)
+	. = ..()
+	atom_storage.max_slots = 8
```

Comment:
Is this really necessary?

https://github.com/tgstation/tgstation/blob/eae17fc5b7037649c003bea20967cb43ed7d5b10/code/game/objects/items/storage/boxes/job_boxes.dm#L18-L25

Reply:
Yep, since that codeblock only works for spawned boxes with the premium internals boxes station trait active

Reply:
Ahhh I missed the early return